src/calls.h

```diff
@@ -45,10 +45,8 @@
 #ifndef GAP_CALLS_H
 #define GAP_CALLS_H
 
-#include <src/gap.h>
-#include <src/gaputils.h>
-#include <src/lists.h>
-#include <src/integer.h>
+#include <src/gaputils.h>    // ARRAY_SIZE
```

Comment:
Maybe remove this comment, if we aren't having them anywhere else?

Reply:
Huh, indeed, not sure how that survived my script.

Reply:
(Or maybe I added it as annotation? Anyway, will remove)